test/for-in-loops.js

```diff
@@ -0,0 +1,13 @@
+const compareResult = require("./util/compare-result");
+const forInLoopsTransform = require("../transforms/for-in-loops");
+
+describe("for-in-loops", () => {
+  compareResult("Iterating through a list", forInLoopsTransform, `
+    ((input, output) ->
+      for item in input
+        output.push item
+    )([1, '2', '3', 'string'], [])
+  `);
```

Comment:
It surprised me that triggered an error in the transform, since it looks quite basic to me, any clue?